deploy/stacks/container.py

```diff
@@ -512,3 +486,11 @@ def set_scheduled_task(
             # security_groups=[security_group],
         )
         return scheduled_task
+
```

Comment:
[Clarification] - Method to simplify env creation. Added `config_location` to the environment of the ECS container

Reply:
I'd say it's more about reducing the duplication of the code 😄